controllers/convoController.js

```diff
@@ -88,10 +88,12 @@ const fetchAllMyConversationsGet = async (req, res) => {
 		if (userData) {
 			conversationList[i] = {
 				...conversationList[i],
+				_id: userData._id,
 				firstName: userData.firstName,
 				lastName: userData.lastName,
 				email: userData.email,
-				avatar: userData.avatar
+				avatar: userData.avatar,
+				userID: undefined
```

Comment:
Instead of `undefined`, since it's technically already defined, a better falsey value may be `null`